### PR TITLE
Update to encfs 1.8

### PR DESCRIPTION
**Pull request by jooray**
Update to latest encfs. It already contains many fixes that were patched here manually. Also removed patch to path to osxfuse, new configure contains the right path already.

### CODE REVIEW

**Comment by mikemcquaid** [comment]
Needs a `needs :cxx11` line to fix the 10.8 failure: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23169/version=mountain_lion/testReport/junit/brew-test-bot/mountain_lion/install_encfs/

**Comment by jooray** [comment]
Added needs :cxx11

**Comment by mikemcquaid** [comment]
Still failing :( May also need a `ENV.cxx11` in the `install` method.

**Comment by jooray** [comment]
Added, we'll see. I sadly have no access to Mountain Lion, so I am kind of blind here.

**Comment by mikemcquaid** [comment]
Hooray it worked!

File: Library/Formula/encfs.rb [hunk 1]
```diff
@@ -5,24 +5,8 @@ class Encfs < Formula
   revision 1
```
Comment (mikemcquaid) [inline Library/Formula/encfs.rb]: Oops, last thing: can you remove this line? Thanks!

**Comment by jooray** [comment]
Amazing!

Line removed ;).

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `encfs 1.8`.

**Pull request closed without merging**